auto-update-dataset/python/repoArchivedCheck.py

```diff
@@ -6,13 +6,15 @@
 # it takes about 2 minutes to check 200 projects
 def main():
     # get unique urls from pr-data.csv, we can also use cmd: `git pull -r ; cut -f1 -d, pr-data.csv | uniq`
-    pr_data_url = r"https://raw.githubusercontent.com/TestingResearchIllinois/idoft/main/pr-data.csv"
+    pr_data_url = r"https://raw.githubusercontent.com/TestingResearchIllinois/idoft/main/py-data.csv"
```

Comment:
How could this work with `py-data.csv` and `pr-data.csv`?

Reply:
I can make it so that the script takes an argument using sys, instead of manually passing the path for each specific file. Would that be appropriate? Otherwise you're going to have to change that every run.

Reply:
Indeed, reading the input would be better than requiring to change the script to run on a different input.